### DISCUSSION TITLE
SDK para react-native

### DISCUSSION THREAD

**Discussion by alexandercamps** [mercadopago/sdk-js#222] [Ideas]
Olá, boa tarde!
Existe um sdk oficial para react-native (android e ios)?

Muito obrigado!

**Comment by icaldana** [comment]
Olá @alexandercamps,

É um desejo nosso termos SDKs para mobile (Android e iOS), então é possível que se tenha novidades sobre o tema ainda em 2025. Sugiro que fique atendo aos nossos canais de comunicação e [developers](https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/landing) para novidades.

Muito obrigado!

**Comment by rafaeloledo** [comment]
@icaldana enquanto n lancam um SDK pra RN, tenho uma pergunta

```js
    fetch(
      "https://api.mercadopago.com/v1/card_tokens?public_key={...}",
      {
        method: "POST",
        headers: {
          "Content-Type": "application/json",
          "X-Idempotency-Key": "...",
        },
        body: JSON.stringify({
          card_number: "4235647728025682",
          cardholder: {
            name: "APRO",
          },
          expiration_month: 11,
          expiration_year: 2030,
          security_code: "123",
        }),
      }
    )
      .then((response) => response.json())
      .then((data) => console.log(data))
      .catch((error) => console.error("Erro:", error))
```

Nao estou conseguindo achar documentacao desse endpoint, posso continuar usando? Imagino que ele seja o endpoint usado por voces.

**Reply by lemazzo** [reply]
Olá, @rafaeloledo 

Não, esse endpoint não deve ser usado. Caso precise de mais ajuda, recomendamos que entre com contato com o [suporte do Mercado Pago](https://www.mercadopago.com.br/developers/pt/support/center/tickets).
